editoast/src/views/pathfinding/mod.rs

```diff
@@ -73,38 +98,28 @@ enum PathfindingError {
     RollingStockNotFound { rolling_stock_id: i64 },
 }
 
-// FIXME: The following routes include "pathfinding" directly in their path, as to not conflict with the
-// other routes. When the routes in mod.rs are moved to utoipa, the paths can be fixed.
-crate::routes! {
-    catenaries::routes(),
-    electrical_profiles::routes(),
-}
-
-crate::schemas! {
-    catenaries::schemas(),
-    electrical_profiles::schemas(),
-}
-
-/// Returns `/pathfinding` routes
-/// TODO: This is not called `routes` because it should be integrated in the `routes!` macro
-pub fn routes_v1() -> impl HttpServiceFactory {
-    web::scope("/pathfinding").service((get_pf, del_pf, create_pf, update_pf))
+#[derive(Clone, Debug, Serialize, Deserialize, Derivative, ToSchema)]
+#[derivative(PartialEq)]
+pub(super) struct PathResponse {
+    #[derivative(PartialEq = "ignore")]
```

Comment:
This struct replaces STDCMPath which already had this behavior, admittedly not ideal, for testing reasons. This pr being big enough I figured it would be better to avoid modifying the tests as much as possible.

Reply:
It wasn't a big change after all. Removed all ignores.